CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # CHANGELOG
 
+## v0.22.0
+
+*November 9, 2022*
+
+This release includes major improvements in making the library compatible with ibc-go v5.0.1. This includes making ibc events compatible and removing the crossing-hellos logic from the connection and channel handshakes.
+
+### BREAKING CHANGES
+
+- Make connection events compatible with ibc-go
+  ([#145](https://github.com/cosmos/ibc-rs/issues/145))
+- Makes channel/packet events compatible with ibc-go
+  ([#146](https://github.com/cosmos/ibc-rs/issues/146))
+- Remove crossing hellos logic from channel handshake
```

Comment:
Looks like we're missing an entry for #156 here.

Reply:
[c16862d](https://github.com/cosmos/ibc-rs/pull/234/commits/c16862defda47ad9af214726167b95e4542f1dd1)